jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile

```diff
@@ -62,6 +62,31 @@ String[] defaultPRLabels = ["product update", productName]
 def installationManifestFile = './inventories/group_vars/all/manifest.yaml'
 String jiraRestApiUrl = "https://issues.jboss.org/rest/api/2"
 
+currentBuild.description = "sourceBranch: ${sourceBranch}\n targetBranch: ${targetBranch}"
+
+String[] jiraLabelsFor(String productName, String targetBranch, def labels = []) {
+    def defaultLabels = [
+            /"unplanned"/,
```

Comment:
Unplanned label is now added to all jiras so they appear in the support sheriff backlog @maleck13 fyi